src/System.Devices.Gpio/System.Devices.Gpio/RaspberryPiDriver.cs

```diff
@@ -0,0 +1,488 @@
+﻿using System.IO;
+using System.IO.MemoryMappedFiles;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace System.Devices.Gpio
+{
+    public unsafe class RaspberryPiDriver : GpioDriver
+    {
+        #region RegisterView        
+
+        [StructLayout(LayoutKind.Explicit)]
+        private struct RegisterView
+        {
+            ///<summary>GPIO Function Select, 6x32 bits, R/W</summary>
+            [FieldOffset(0x00)]
+            public fixed UInt32 GPFSEL[6];
+
+            /////<summary>GPIO Function Select 0, 32 bits, R/W</summary>
+            //[FieldOffset(0x00)]
```

Comment:
Why are all these commented out?

Reply:
Those are not needed because we are using the fixed array version of those registers. I just left them there as documentation but I can actually remove them.

Reply:
Removing those make easy to read the sources.